docs/operation-guides/15-backup-volume-snapshots.md

```diff
@@ -54,6 +54,10 @@ This creates a new `pvc-restored` PVC with pre-populated data from the snapshot.
 
 You can also create a CronJob to handle taking volume snapshots periodically. A sample CronJob definition that includes the required ServiceAccount and roles looks as follows:
 
+<details>
+<summary>Expand sample code</summary>
+<p>
```

Comment:
I'm not sure if that is necessary as I suppose it will be collapsed on the website one way or another. We rather use these tabs if we have two or more items.

Reply:
Do we collapse code blocks? I didn't know. That would be great!
I wanted to try this method out here, but the preview didn't render it collapsed anyway.

Reply:
It will be collapsed only on GitHub but will show the full snippet on the website anyways.

Reply:
Maybe I'll create a ticket requesting that the website can display the code collapsed too :D